code/obj/item/retribution.dm

```diff
@@ -0,0 +1,233 @@
+/obj/item/sword_core
+	name = "SWORD core"
+	desc = "An incredibly advanced power core created by the Syndicate."
+	icon = 'icons/misc/retribution/SWORD_loot.dmi'
+	icon_state = "engine_core"
+	flags = FPRINT | TABLEPASS | CONDUCT | ONBELT
+	w_class = 2.0
+	throw_speed = 4
+	throw_range = 20
+	is_syndicate = 1
+	contraband = 5
+
+/obj/item/syndicate_destruction_system
+	name = "Syndicate Destruction System"
+	desc = "An unfinished melee weapon, the blueprints for which have been plundered from a raid on a now-destroyed Syndicate base."
```

Comment:
I suggest adding to the description that the weapon requires a unique engine for it to work, so as not to confuse people.

Reply:
Noted, will do that later as well.

Reply:
Done.